src/components/SlackButton/index.js

```diff
@@ -11,9 +11,10 @@ import {
 import slackLogo from '../../assets/slack-logo.png';
 import arrow from '../../assets/arrow.png';
 
+const clientId = process.env.REACT_APP_CLIENT_ID;
+const redirectURI = `${document.location.origin}/slack`;
```

Comment:
Nice drying Noble! Btw, why not extract out scope as well?

Reply:
@ropeks the scopes for each of the button is different.
-  Sign in with Slack Button: 
`const scope = 'identity.basic,identity.email,identity.team,identity.avatar';`
- Add to Slack button: 
`const scope = 'incoming-webhook,commands';`

Reply:
@nobioma1 thanks for clearing that up!